cloudinit/net/sysconfig.py

```diff
@@ -99,13 +99,18 @@ def drop(self, key):
     def __len__(self):
         return len(self._conf)
 
+    def str_skipped(self, key, val):
+        return False
```

Comment:
why is this defined a second time here, and it's unconditionally returning `False`
is this a debug thing?

Reply:
I guess the fake function needs to exist in order to to_string() function exist. But then it's redefined on the class NetInterface() that inherits the Class ConfigMap()

Reply:
> why is this defined a second time here, and it's unconditionally returning `False`
> is this a debug thing?

`ConfigMap.to_string()` calls `self.str_skipped(...)`, so it needs to be defined on `ConfigMap` with its default behaviour (which is to skip no strings). `NetInterface` inherits from `ConfigMap`, inheriting its `to_string` method and overriding its `str_skipped` method (to skip `TYPE=Vlan` lines).